include "atoms/tenfour.thrift"

union ContentAtom {
  1: tenfour.Content tenfour
}
